app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -90,6 +98,26 @@ class AddDeadlineActivity : AppCompatActivity() {
         }
 
         initializePlacesAutocomplete()
+
+        // Update the groups owned by the user to be able to select them
+        // in the Dialog
```

Comment:
This comment is useless... This is already documented in the Javadoc of the method.